README.md

```diff
@@ -1 +1,51 @@
-# Review-past-project
\ No newline at end of file
+# Review-past-project
+
+
+## Built With
+- Html
+- CSS
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+html/css/github workflow knowledge
+
+### Setup
+Set up and configure git on your system.
+
+### Install
+```
+git clone https://github.com/Kalunge/portfolio-project.git
+
+```
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@kalunge](https://github.com/kalunge)
+- Twitter: [@kalunge](https://twitter.com/titus_muthomi)
+- LinkedIn: [kalunge](https://linkedin.com/in/titus_muthomi)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
fix this broken link

Reply:
- I think your review comment should be polite. It should also be a suggestion instead of making it look like you want to give orders. People's perceptions are different and you have to make sure your review comment is acceptable to all.
It could be something like this :point_right: "I think this link is broken. Leaving a broken link in your README.md file might generate a negative feeling to your audience". What do you think?